pkg/adaptor/cloud/azure/provider.go

```diff
@@ -419,7 +419,8 @@ func (p *azureProvider) getVMParameters(instanceSize, diskName, b64EncData strin
 		securityProfile = &armcompute.SecurityProfile{
 			SecurityType: to.Ptr(armcompute.SecurityTypesConfidentialVM),
 			UefiSettings: &armcompute.UefiSettings{
-				SecureBootEnabled: to.Ptr(true),
+				// Turn off, we want to do measured boot. Secure boot brings no additional security.
+				SecureBootEnabled: to.Ptr(false),
```

Comment:
Can we use a config flag ( like `disable-cvm` ) for this setting with default set to false? I'm thinking about some scenarios where this setting could be a checklist item.

Reply:
I've added a flag `enable-secure-boot` that is false by default.

Reply:
@bpradipt could you check if its working out for you this way?